lib/phase/structural.js

```diff
@@ -12,6 +12,10 @@ const SCHEMA_CONFIG = require('screwdriver-data-schema').config.base.config;
 function checkAdditionalRules(data) {
     const { jobs } = data;
 
+    if (!jobs) {
+        throw new Error('Pipeline has no jobs');
+    }
```

Comment:
Current implementation performs structural validation after merging pipeline template config into to pipeline config.
Why/when would `jobs` be `undefined`?

Reply:
that that check happens even when you are not using pipeline template